src/resources/postcss/components/_events-bar.pcss

```diff
@@ -43,8 +44,8 @@
 		}
 	}
 
-	.tribe-events-c-events-bar__form {
-		display: none;
+	.tribe-events-c-events-bar__search {
+		/*display: none;*/
```

Comment:
Does this need to get removed at some point (maybe a todo note would be helpful here)?

Reply:
My bad, I was checking the inputs on mobile viewports - we'll have a toggle that will take care of the visibility of the form. Seems that I commited the changes when renaming the selector